dist/css/other_assets.css

```diff
@@ -5,3 +5,7 @@
     padding: 0;
     background: url('/assets/original/equipment-dude.jpg') no-repeat;
 }
+
```

Comment:
The option to change text color doesn't exist in semantic-ui: https://github.com/Semantic-Org/Semantic-UI/issues/1885
So I'll go for elm-css.